pkg/restic/commands.go

```diff
@@ -331,7 +328,7 @@ func (w *ResticWrapper) stats(snapshotID string) ([]byte, error) {
 		args = append(args, snapshotID)
 	}
 	args = w.appendMaxConnectionsFlag(args)
-	args = append(args, "--quiet", "--json", "--mode", "raw-data")
+	args = append(args, "--quiet", "--json", "--mode", "--no-lock", "raw-data")
```

Comment:
You can't inject `--no-lock` between `--mode raw-data`. Here, `raw-data` is the input of `--mode` flag.